saleor/payment/gateways/stripe/stripe_api.py

```diff
@@ -236,6 +244,7 @@ def capture_payment_intent(
                 amount_to_capture=amount_to_capture,
                 api_key=api_key,
                 stripe_version=STRIPE_API_VERSION,
+                expand=["payment_method"],
```

Comment:
Why do you need it?

Reply:
Stripe normally returns PaymentIntent with PaymentMethod as its ID. To get the full StripeObject, we need to pass the expand parameter.

Reply:
I understand but I don't know why you need this for the `capture payment intent` function. It is called in  `StripePlugin.capture_payment` so it seems that this change is not related to your feature

Reply:
`capture_payment`, which calls `capture_payment_intent`, also calls `get_payment_method_details`, which assumes that `payment_method` is expanded. If `payment_method` isn't needed there, I suggest two alternative solutions: 
* check inside `get_payment_method_details` if `payment_method` is string and if so - return empty metadata
* split it to two functions: `get_payment_method_details` and `get_payment_method_details_with_metadata`